lib/board.rb

```diff
@@ -11,4 +11,23 @@ def initialize
       [nil, nil, nil]
     ]
   end
+
+  def make_move(piece, position)
+    row_and_col = position_to_row_and_col position
+    row = row_and_col[0]
+    col = row_and_col[1]
+
+    @board[row][col] = piece
+  end
+
+  def position_to_row_and_col(position)
+    # convert to zero-based numbering
```

Comment:
When I need to leave a comment in my code I think of it as a sign that I could employ better variable names or extract a method to name the concept I was adding a comment to explain.
https://refactoring.guru/extract-method